pkg/codegen/internal/test/testdata/kubernetes-operator.pp.cs

```diff
@@ -0,0 +1,282 @@
+                            {
+                                Name = "pulumi-kubernetes-operator",
+                            },
+                        },
+                        Containers = 
+                        {
+                            new Kubernetes.Core.Inputs.ContainerArgs
+                            {
+                                Name = "pulumi-kubernetes-operator",
+                                Image = "pulumi/pulumi-kubernetes-operator:v0.0.2",
+                                Command = 
+                                {
+                                    "pulumi-kubernetes-operator",
+                                },
+                                Args = 
+                                {
+                                    "--zap-level=debug",
+                                },
+                                ImagePullPolicy = "Always",
+                                Env = 
+                                {
+                                    new Kubernetes.Core.Inputs.EnvVarArgs
+                                    {
+                                        Name = "WATCH_NAMESPACE",
+                                        ValueFrom = new Kubernetes.Core.Inputs.EnvVarSourceArgs
+                                        {
+                                            FieldRef = new Kubernetes.Core.Inputs.ObjectFieldSelectorArgs
+                                            {
+                                                FieldPath = "metadata.namespace",
+                                            },
+                                        },
+                                    },
+                                    new Kubernetes.Core.Inputs.EnvVarArgs
+                                    {
+                                        Name = "POD_NAME",
+                                        ValueFrom = new Kubernetes.Core.Inputs.EnvVarSourceArgs
+                                        {
+                                            FieldRef = new Kubernetes.Core.Inputs.ObjectFieldSelectorArgs
+                                            {
+                                                FieldPath = "metadata.name",
+                                            },
+                                        },
+                                    },
+                                    new Kubernetes.Core.Inputs.EnvVarArgs
+                                    {
+                                        Name = "OPERATOR_NAME",
+                                        Value = "pulumi-kubernetes-operator",
+                                    },
+                                },
+                            },
+                        },
+                    },
+                },
+            },
+        });
+        var pulumi_kubernetes_operatorRole = new Kubernetes.Rbac.authorization.k8s.io.v1.Role("pulumi_kubernetes_operatorRole", new Kubernetes.Rbac.authorization.k8s.io.v1.RoleArgs
```

Comment:
@mikhailshilkov Looks like the dotnet program gen is not consuming the `namespaces` mapping here